src/PowerShell/Microsoft.WinGet.Client/Crescendo.json

```diff
@@ -0,0 +1,127 @@
+﻿{
+    "$schema": "https://aka.ms/PowerShell/Crescendo/Schemas/2021-11",
+    "Commands": [
+        {
+            "Verb": "Get",
+            "Noun": "WinGetVersion",
+            "Platform": [
+                "Windows"
+            ],
+            "OriginalName": "winget.exe",
+            "OriginalCommandElements": [
+                "--version"
+            ]
+        },
+        {
+            "Verb": "Enable",
+            "Noun": "WinGetLocalManifest",
```

Comment:
Maybe the problem is that authoring it in Crescendo is difficult, but I know I wanted this to be something more like `Enable-WinGetSetting LocalManifestFiles` rather than having a cmdlet per setting.

I think I would rather not have the cmdlet at all than have this if we have to choose.

Reply:
A possible solution could be giving `Enable-WinGetSetting` a `Parameter(Mandatory, Position = 0)` parameter of something like `-Name` and then invoking `winget settings --enable $Name` which is possible in Crescendo.